$OpenBSD: patch-GNUmakefile,v 1.3 2004/04/29 03:49:51 dons Exp $

Make the install a bit more standard.
Don't try and install .ps docs

--- GNUmakefile.orig	2004-04-04 02:57:00.000000000 +1000
+++ GNUmakefile	2004-04-24 13:10:56.000000000 +1000
@@ -87,15 +87,14 @@ installbin:	all
 	$(INSTALL) -d $(DESTDIR)/$(bindir)
 	$(INSTALL) darcs $(DESTDIR)/$(bindir)
 
-	$(INSTALL) -d $(DESTDIR)/$(sysconfdir)/bash_completion.d/
-	$(INSTALL) -m 0644 darcs_completion $(DESTDIR)/$(sysconfdir)/bash_completion.d/darcs
+	$(INSTALL) -d $(DESTDIR)/$(datadir)/darcs/bash_completion.d/
+	$(INSTALL) -m 0644 darcs_completion $(DESTDIR)/$(datadir)/darcs/bash_completion.d/darcs
 
 	$(INSTALL) -d $(DESTDIR)/$(mandir)/man1
 	$(INSTALL) -m 0644 darcs.1 $(DESTDIR)/$(mandir)/man1/
 
-installdocs:	darcs.ps manual/index.html
+installdocs:
 	$(INSTALL) -d $(DESTDIR)/$(datadir)/doc/darcs/manual
-	$(INSTALL) -m 0644 darcs.ps $(DESTDIR)/$(datadir)/doc/darcs
 	$(INSTALL) -m 0644 manual/* $(DESTDIR)/$(datadir)/doc/darcs/manual
 
 	$(INSTALL) -d $(DESTDIR)/$(datadir)/doc/darcs/examples
@@ -103,15 +102,15 @@ installdocs:	darcs.ps manual/index.html
 	$(INSTALL) -m 0644 zsh_completion_old $(DESTDIR)/$(datadir)/doc/darcs/examples/
 
 installserver:	darcs_cgi darcs-createrepo
-	$(INSTALL) -d $(DESTDIR)/$(libexecdir)/cgi-bin
-	$(INSTALL) darcs_cgi $(DESTDIR)/$(libexecdir)/cgi-bin/darcs
+	$(INSTALL) -d $(DESTDIR)/$(libexecdir)/darcs
+	$(INSTALL) darcs_cgi $(DESTDIR)/$(libexecdir)/darcs/darcs.cgi
 
 	$(INSTALL) -d $(DESTDIR)/$(sbindir)
 	$(INSTALL) darcs-createrepo $(DESTDIR)/$(sbindir)
 
-	$(INSTALL) -d $(DESTDIR)/$(sysconfdir)/darcs
-	test -e $(DESTDIR)/$(sysconfdir)/darcs/cgi.conf || \
-	    $(INSTALL) -m 0644 cgi.conf $(DESTDIR)/$(sysconfdir)/darcs/cgi.conf
+	$(INSTALL) -d $(DESTDIR)/$(datadir)/darcs
+	test -e $(DESTDIR)/$(datadir)/darcs/cgi.conf || \
+		$(INSTALL) -m 0644 cgi.conf $(DESTDIR)/$(datadir)/darcs/cgi.conf
 
 dist:		darcs
 	./darcs dist --dist-name darcs-`./darcs --version`
